homework-submission/week1-h.w/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import "./App.css";
+import Item from "./Item.js";
+import ItemData from "./ItemData.js";
+import Header from "./Header";
+
+class App extends Component {
+  render() {
+    const Data = ItemData.map(Data => (
+      <Item text={Data.text} deadline={Data.deadline} />
+    ));
+    console.log(Data);
```

Comment:
don't leave the console.log in your code when you push it

Reply:
i usually remove it before pushing. my bad